catroid/src/main/java/org/catrobat/catroid/stage/PreStageActivity.java

```diff
@@ -240,6 +245,33 @@ public void onClick(DialogInterface dialog, int id) {
 			task.execute();
 		}
 
+		if ((requiredResources & Brick.NETWORK_CONNECTION) > 0) {
```

Comment:
Please change this condition to (requiredResources & Brick.NETWORK_CONNECTION) != 0
As this is a bitwise AND between two integers, it could get negative as we may reach the 32nd bit with future changes